tests/test_pango.py

```diff
@@ -137,3 +137,20 @@ def test_tabs_replace() -> None:
     pangocairocffi.show_layout(context, layout)
     surface.finish()
     assert compare_SVGObject_with_PangoText(temp_pango_text, filename)
+
+
+def check_t2s() -> None:
```

Comment:
**Question** i'm pretty sure it's intended, but I can't see the reason. Why is it `check_t2s` and not `test_t2s`?

Reply:
That needs to be `test`.